### PR TITLE
[16.0][IMP] sign_oca: Add General settings menu item

### PR DESCRIPTION
**Pull request by victoralmau**
FWP from 15.0: https://github.com/OCA/sign/pull/68

Add General settings menu item

Please @pedrobaeza and @pilarvargas-tecnativa can you review it?

@Tecnativa TT41745

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: sign_oca/__manifest__.py [hunk 1]
```diff
@@ -14,10 +14,10 @@
         "security/security.xml",
         "views/menu.xml",
         "data/data.xml",
-        "wizards/res_config_settings_views.xml",
         "data/ir_sequence_data.xml",
         "wizards/sign_oca_template_generate.xml",
         "wizards/sign_oca_template_generate_multi.xml",
+        "views/res_config_settings_views.xml",
```
Comment (pedrobaeza) [inline sign_oca/__manifest__.py]: Why changing it to `views` and `models`? It's a transient one.

Comment (victoralmau) [reply sign_oca/__manifest__.py]: Because even if they are transient `res.config.settings` they have always been set in models and in views as exception (because they are not really a wizard).

Comment (pedrobaeza) [reply sign_oca/__manifest__.py]: I would say that's a mistake, but being in the other place, let's not change it.

File: sign_oca/views/menu.xml [hunk 2]
```diff
@@ -13,10 +13,11 @@
         sequence="16"
         groups="sign_oca.sign_oca_group_user"
     />
-    <record model="ir.ui.menu" id="sign_oca_settings_menu">
-        <field name="name">Settings</field>
-        <field name="sequence" eval="90" />
-        <field name="parent_id" ref="sign_oca_root_menu" />
-        <field name="groups_id" eval="[(6,0,[ref('sign_oca.sign_oca_group_admin')])]" />
-    </record>
+    <menuitem
```
Comment (pedrobaeza) [inline sign_oca/views/menu.xml]: I don't get why changing this, and more in this commit that is unrelated.

Comment (victoralmau) [reply sign_oca/views/menu.xml]: I honestly don't understand these comments, it's something that has already been changed in v15 in the corresponding PR, but no problem, it's already changed.

**Review by pedrobaeza — APPROVED** [review]
In v15 it was slipped into the PR, but only for one item, so it was not so critical, but changes should be ordered for the commit history, and we have to avoid cosmetic changes most of the times.

/ocabot merge minor
